### PR TITLE
Add tooltip to clean search button

### PR DESCRIPTION
**Pull request by zdubzdab**

### CODE REVIEW

**Comment by zdubzdab** [comment]
screenshot from 2016-03-18 12 47 33

File: app/assets/stylesheets/welcome.scss [hunk 1]
```diff
@@ -22,21 +22,48 @@
...
+  position:absolute;
+  left: -65px;
+  top: -32px;
+  color:#111;
+  border:1px solid #DCA;
+  background:#fffAF0;
   }
+
+a#clean_search_field span::after {
+  content: " ";
```
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: Why do you need after pseudo class ? Is it possible to complete do it without after?

Comment (zdubzdab) [inline app/assets/stylesheets/welcome.scss]: I added  after class in order to have arrow in bottom of message

Comment (zdubzdab) [inline app/assets/stylesheets/welcome.scss]: http://www.w3schools.com/css/css_tooltip.asp

Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: cool =)

**Pull request merged**